### PR TITLE
Implement .env for config variables

### PR DESCRIPTION
**Pull request by squidicuzz**
Adds `.env` file for configuration variables.

image

### CODE REVIEW

File: metrix-explorer-ui/plugins/websocket.js [hunk 1]
```diff
@@ -3,8 +3,9 @@ import io from 'socket.io-client'
 
 export default function ({ app }) {
   let ws = io(
-    'ws:' + process.env.metrixinfoWSBase,
-    { transports: ['websocket'] }
+    process.env.hostprot + '//' + process.env.metrixinfoWSBase,
+    //'ws:' + process.env.metrixinfoWSBase,
+    //{ transports: ['websocket'] }
```
Comment (squidicuzz) [inline metrix-explorer-ui/plugins/websocket.js:8]: This is defaulting to port 80... or `ws://` ... How to use `wss://` ?

**Comment by squidicuzz** [comment]
Updated all 3 projects to generate and use better configuration default loading.

File: metrix-explorer-ui/plugins/websocket.js (hunk 1 above)
Comment (nibbles83) [reply metrix-explorer-ui/plugins/websocket.js:8]: This probably needs its own var for the prefix depending if you want to use SSL or not.
using wss: is the only change from the original code to enable secure comms using the websocket.
process.env.hostprot is https according to the env file, so not sure how swapping ws to https will work.

Comment (squidicuzz) [reply metrix-explorer-ui/plugins/websocket.js:8]: It was force using `ws` for some reason.  I could not get `wss` to work when using ssl.

I got it to work using `https` though..  In the `.env` I declare the protocol to use as `https:`
image

**Pull request merged**